apache-airflow-mixin/dashboards.libsonnet

```diff
@@ -0,0 +1,115 @@
+    local tags = this.config.dashboardTags;
+    local uid = g.util.string.slugify(this.config.uid);
+    local vars = this.grafana.variables;
+    local annotations = this.grafana.annotations;
+    local refresh = this.config.dashboardRefresh;
+    local period = this.config.dashboardPeriod;
+    local timezone = this.config.dashboardTimezone;
+
+    {
+      // Apache Airflow overview dashboard
+      'apache-airflow-overview.json':
+        g.dashboard.new(prefix + ' overview')
+        + g.dashboard.withDescription('Dashboard providing an overview of Apache Airflow DAGs, tasks, and scheduler performance.')
+        + g.dashboard.withPanels(
+          g.util.panel.resolveCollapsedFlagOnRows(
+            g.util.grid.wrapPanels(
+              [
+                this.grafana.rows.apacheAirflowOverview,
+                this.grafana.rows.apacheAirflowSchedulerDetails,
+              ]
+            )
+          )
+        )
+        + root.applyCommon(
+          vars.multiInstance + [
+            g.dashboard.variable.query.new('dag_id')
+            + g.dashboard.variable.query.withDatasourceFromVariable(vars.datasources.prometheus)
+            + g.dashboard.variable.query.queryTypes.withLabelValues('dag_id', 'airflow_dagrun_duration_success_sum{job=~"$job", instance=~"$instance"}')
+            + g.dashboard.variable.query.generalOptions.withLabel('DAG ID')
+            + g.dashboard.variable.query.selectionOptions.withMulti(true)
+            + g.dashboard.variable.query.selectionOptions.withIncludeAll(true, '.+')
+            + g.dashboard.variable.query.refresh.onLoad()
+            + g.dashboard.variable.query.refresh.onTime(),
+
+            g.dashboard.variable.query.new('task_id')
+            + g.dashboard.variable.query.withDatasourceFromVariable(vars.datasources.prometheus)
+            + g.dashboard.variable.query.queryTypes.withLabelValues('task_id', 'airflow_task_finish_total{job=~"$job", instance=~"$instance"}')
+            + g.dashboard.variable.query.generalOptions.withLabel('Task')
+            + g.dashboard.variable.query.selectionOptions.withMulti(true)
+            + g.dashboard.variable.query.selectionOptions.withIncludeAll(true, '.+')
+            + g.dashboard.variable.query.refresh.onLoad()
+            + g.dashboard.variable.query.refresh.onTime(),
+
+            g.dashboard.variable.query.new('state')
+            + g.dashboard.variable.query.withDatasourceFromVariable(vars.datasources.prometheus)
+            + g.dashboard.variable.query.queryTypes.withLabelValues('state', 'airflow_task_finish_total{job=~"$job", instance=~"$instance"}')
+            + g.dashboard.variable.query.generalOptions.withLabel('State')
+            + g.dashboard.variable.query.selectionOptions.withMulti(true)
+            + g.dashboard.variable.query.selectionOptions.withIncludeAll(true, '.+')
+            + g.dashboard.variable.query.refresh.onLoad()
+            + g.dashboard.variable.query.refresh.onTime(),
```

Comment:
These three template variables aren't showing any values in the sample-app. Missing load-gen perhaps?